README.md

```diff
@@ -12,70 +12,193 @@ Airport Challenge
+*Test 3* - Return an error message when capacity is inputted as below 0  
+*Test 4* - Return an error message when capacity is inputted as null  
+*Test 5* - Return an error message when capacity is inputted as a non-numerical string  
+  
+To note: For tests 2.3 - 2.5, I wanted to throw an error rather than set the capacity as a string, however I then couldn't run the tests.  
+
+## User Story 3  
+As an air traffic controller  
+To ensure safety  
+I want to prevent landing when the airport is full  
+
+**Domain Model 3**
+
+| Objects | Properties                   | Messages          | Outputs |
+| ------- | -------------------------    | ----------------- | ------- |
+| Airport |planesAtAirport @Array[@Plane]| planeLands(@Plane)| @Void   |
+|         |airportName@String            |                   |         |
+|         |airportCapacity @Integer      | atCapacity()      |@Boolean |
+| Plane   |plane id @String              |                   |         |
+
+**Tests**  
+
+*Test 1* - When the length of planesAtAirport and airportCapacity are the same, the test returns true  
+*Test 2* - When the length of planesAtAirport and airportCapacity are the same (1), atCapacity returns true  
+*Test 3* - When the length of planesAtAirport and airportCapacity are different, atCapacity returns false  
+*Test 4* - When atCapacity is true, planeLands returns an error message  
+*Test 5* - When atCapacity is true, planeLands doesn't change the length of planesAtAirport array   
+*Test 6* - When atCapacity is false, planeLands increases length of planesAtAirport array by 1  
+
+## User Story 4  
+As an air traffic controller  
+So I can get passengers on the way to their destination  
+I want to instruct the airport to let a plane take off and confirm that it is no longer in the airport  
+
+**Domain Model 4**  
+
+| Objects | Properties                   | Messages                          | Outputs |
+| ------- | -------------------------    | -----------------------------     | ------- |
+| Airport |planesAtAirport @Array[@Plane]| planeLands(@Plane)                | @Void   |
+|         |airportName@String            |                                   |         |
+|         |airportCapacity @Integer      | atCapacity()                      | @Boolean|
+|         |                              | planeTakesOff()                   | @String |
+|         |                              | isPlaneAtAirport(@Plane)          | @Boolean|
+| Plane   |planeID @String               |                                   |         |
+
+**Tests**  
+
+*Test 1* - Length of planesAtAirport array at the Airport decreases by 1 when planeTakesOff  
+*Test 2* - Confirm if plane is in planesAtAirport using isPlaneAtAirport returns true  
+*Test 3* - Confirm if plane is in planesAtAirport using isPlaneAtAirport returns false  
+*Test 4* - When planeTakesOff is called and isPlaneAtAirport returns true, length of planesAtAirport decreases by 1  
```

Comment:
Do I need this test (and similar ones below) where I check the length of planesAtAirport again as I've created a new method/changed something? Or would the original passing (4.1) be enough?